### PR TITLE
ROX-16950: Add more known namespaces to default policy exclusion list

### PR DESCRIPTION
**Pull request by theencee**
## Description

More exclusions from this list: https://drive.google.com/file/d/12li-za7ElioYY7bIq64hDQOn_w1SpE_k/view?usp=sharing

I also added a migration for [this policy change](https://github.com/stackrox/stackrox/pull/5993) which just fixes a grammatical error in the description.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: migrator/migrations/m_184_to_m_185_more_policy_migrations/migration_test.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type categoriesMigrationTestSuite struct {
```
Comment (clickboo) [inline migrator/migrations/m_184_to_m_185_more_policy_migrations/migration_test.go:22]: naming?

Comment (theencee) [reply migrator/migrations/m_184_to_m_185_more_policy_migrations/migration_test.go]: copy and paste 🤦

File: migrator/migrations/m_185_to_m_186_more_policy_migrations/migration.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+		},
+		{
+			FieldsToCompare: fieldsToCompareForExclusions,
+			PolicyFileName:  "root_user.json",
+		},
+		{
+			FieldsToCompare: fieldsToCompareForExclusions,
+			PolicyFileName:  "secret_env.json",
+		},
+		// TODO: Add policy exclusions once they are known
```
Comment (dashrews78) [inline migrator/migrations/m_185_to_m_186_more_policy_migrations/migration.go:97]: Do we need to do anything with this TODO or delete it?

Comment (theencee) [reply migrator/migrations/m_185_to_m_186_more_policy_migrations/migration.go]: Dang it. Deleted

**Review by clickboo — APPROVED** [review]
Lgtm. Disclaimer:I haven't checked each and every exclusion entry for correctness.

**Review by ksurabhi91 — APPROVED** [review]
LGTM

**Pull request merged**